tasks/ci.rake

```diff
@@ -243,9 +260,10 @@ end
 def beaker_suite(type)
```

Comment:
Can we just make the retry optional here? E.g.
```
def beaker_suite(type, options = {})
end
```

then you can do
```
rescue RuntimeError => e
  raise e unless options[:retry]
  // retry code
end
```

Reply:
Thanks! I'll look at that option for the next release. In theory, we might be able to ship this tomorrow; makes me reluctant to have to change things right now.